src/components/MainContent/index.js

```diff
@@ -23,11 +26,10 @@ function MainContent() {
     };
 
     const onFavoriteClick = () => {
-        // on click look up local storage item
-        // add new coin id to local storage item
-        const newFavorites = [...favorites, data.id];
-        localStorage.setItem("favoriteCoinIds", newFavorites);
-        dispatch(updateFavorites(newFavorites));
+        // dispatch addNewFavorite with argument of coin id saved in state in this component - powering the value of the input
+        dispatch(addNewFavorite(searchedCoin));
+        dispatch(importSavedFavorites());
+        // window.alert("added to favorites!");
     };
 
     // TODO
```

Comment:
This sort of TODO stuff i only added cuz its not my project 😆 & it was a moment of need for a reminder sort of comment. You shouldn't keep these sorts of comments in your code. This is a prime example of what should be a Trello ticket/task